weblab_cg/templates/chaste/normal_model.cpp

```diff
@@ -0,0 +1,157 @@
+//! @file
+//! 
+//! This source file was generated from CellML.
+//! 
+//! Model: {{model_name}}
+//! 
+//! Processed by webalab_cg - CellML Tools in Python: https://github.com/ModellingWebLab/weblab-cg
+//!     (translators: , webalab_cg: , options: normal)
+//! on {{generation_date}}
+//! 
+//! <autogenerated>
+
+#include "{{file_name}}.hpp"
+#include <cmath>
+#include <cassert>
+#include <memory>
+#include "Exception.hpp"
+#include "OdeSystemInformation.hpp"
+#include "RegularStimulus.hpp"
+#include "HeartConfig.hpp"
+#include "IsNan.hpp"
+#include "MathsCustomFunctions.hpp"
+    {%- if default_stimulus_equations.membrane_stimulus_current_duration is defined %}
+
+    boost::shared_ptr<RegularStimulus> {{class_name}}FromCellML::UseCellMLDefaultStimulus()
```

Comment:
Could `class_name` not be the full name, rather than just a prefix?

Reply:
sure